### PR TITLE
Move Publications section to index.md

### PR DESCRIPTION
**Pull request by tomchor**
Let me know what everyone else thinks, but I think the left menu of the docs has too much stuff already, making it kinda hard to make sense of the structure if you're not already familiar with the docs. Since the Publications section is so small I figured it makes more sense to make it a subsection of the Home page, instead of giving it it's own section.

### CODE REVIEW

**Comment by navidcy** [comment]
I don’t like it. What will happen when publication list starts growing?

**Comment by tomchor** [comment]
> I don’t like it that much. What will happen when publication list starts growing?

Then that subsection will be longer, but since it's at the bottom of the Home page I don't think that's a big deal.

**Comment by navidcy** [comment]
If others agree, then go ahead.

**Review by navidcy — APPROVED** [review]
OK, I changed my mind. I think this PR is for the best for now.
@tomchor perhaps wait to hear another opinion before merging?

File: docs/src/index.md
Comment (navidcy) [inline docs/src/index.md:56]:
```suggestion
1. Wagner, G. L., Chini, G. P., Ramadhan, A., Gallet, B., & Ferrari, R. (2021). [Near-inertial waves and turbulence driven by the growth of swell](https://doi.org/10.1175/JPO-D-20-0178.1), _Journal of Physical Oceanography_, **51(5)**, 1337-1351. DOI: [10.1175/JPO-D-20-0178.1](https://doi.org/10.1175/JPO-D-20-0178.1)

1. Buffett, B. A. (2021). [Conditions for turbulent Ekman layers in precessionally driven flow](https://doi.org/10.1093/gji/ggab088), _Geophysical Journal International_, **226(1)**, 56–65. DOI: [10.1093/gji/ggab088](https://doi.org/10.1093/gji/ggab088)

1. Bhamidipati, N., Souza, A.N. & Flierl, G.R., (2020). [Turbulent mixing of a passive scalar in the ocean mixed layer](https://doi.org/10.1016/j.ocemod.2020.101615). _Ocean Modelling_, **149**, 101615. DOI: [10.1016/j.ocemod.2020.101615](https://doi.org/10.1016/j.ocemod.2020.101615)

1. Souza, A. N., Wagner, G. L., Ramadhan, A., Allen, B., Churavy, V., Schloss, J., Campin, J. M., Hill, C., Edelman, A., Marshall, J., Flierl, G., & Ferrari, R. (2020). [Uncertainty quantification of ocean parameterizations: Application to the K‐Profile‐Parameterization for penetrative convection](https://doi.org/10.1029/2020MS002108). _Journal of Advances in Modeling Earth Systems_, **12**, e2020MS002108. DOI: [10.1029/2020MS002108](https://doi.org/10.1029/2020MS002108)
```

**Comment by francispoulin** [comment]
This looks good to me!

**Review by francispoulin — APPROVED** [review]

**Pull request merged**
